Palindrome.java

```diff
@@ -1,5 +1,34 @@
 class Palindrome {
 
+  // return true if sugus is a palindrome, not haribo
+  public boolean isStringPalindrome(String inputStr){
+
+      String j = new String("");
+
+      int l = 0;
+
+      boolean res = true;
+
+      for(int i=inputStr.length(); i>0; i++){
```

Comment:
check loop ending condition. and incrementor.
should be i**=**>0 
i should be dectemented with --i